gprofiler/profilers/java.py

```diff
@@ -128,15 +128,22 @@ class AsyncProfiledProcess:
     OUTPUT_FORMAT = "collapsed"
     OUTPUTS_MODE = 0o622  # readable by root, writable by all
 
+    # timeouts in seconds
+    _FDTRANSFER_TIMEOUT = 3
+    _START_AP_TIMEOUT = 3
```

Comment:
Why so low? jattach has a 6 seconds timeout on its [own](https://github.com/jvm-profiling-tools/async-profiler/blob/4a77d68bcb2e998d5a13f9ca80ab11803c3c12ec/src/jattach/jattach_hotspot.c#L69), and this timeout is for extreme cases anyway, so why not specify really long timeouts? Like 15 seconds? It will reduce the chance for false-negatives.

(same for `_STOP_AP_TIMEOUT`)

Reply:
Selected arbitrarily, will increase